torch/csrc/autograd/variable.cpp

```diff
@@ -93,14 +113,21 @@ ViewInfo ViewInfo::chain(
         // first call site is indeed in **forward** pass when we refresh
         // `grad_fn` triggered by inplace update. Search Note [View + Inplace
         // update for view tensor] to for the call site.
+        // TODO: Test for this case!
+        auto error_msg =
+            ("This view is the output of a function that returns multiple views."
+             "Such functions do not allow the output views to be modified inplace."
+             "You should replace the inplace operation by an out-of-place one");
```

Comment:
We can run into this case now without doing in-place over view right? Maybe we shouldn't error here?

Reply:
cc @albanD can I get your thoughts here?

My understanding is that this branch is now only called if:
* We're chaining two views
* Current view has a `view_func()` but parent view does not
* The base tensor does not support `as_strided()`, so we can't cover the gap of the parent view's missing `view_func()`

Note that multiple-view ops like `unbind()` now have `view_func()`s, so the error message is certainly wrong when it mentions the view being an output of a function that returns multiple views.

Can we just error for both `view_func()` / `rev_view_func()` explaining that we can't chain views here because `as_strided()` isn't supported? I expect this to be fairly rare / maybe impossible to hit, as it'd need to be a subclass that doesn't support `as_strided()`, which we should unconditionally generate `view_func()`s for now.

Reply:
For posterity: changed error msg to describe the situation literally for both forward / reverse